cymetric/convenient_interface.py

```diff
@@ -0,0 +1,121 @@
+from pyne import nucname
+import pandas as pd
+import numpy as np
+import cymetric as cym
+
+def GetTransactionTimeSeries(db, sender='All', receiver='All', *xargs):
+
+  
+  nuc_list = []
+  for inx, nuc in enumerate(xargs):
+    nuc_list.append(nucname.id(nuc))
+
+  #initiate evaluation
+  evaler = cym.Evaluator(db)
+  
+  # get transation & Agent tables
+  trans = evaler.eval('Transactions')
+  agents = evaler.eval('AgentEntry')
+  
+# build 2 table for SenderId and ReceiverId
+  # get receiver
+  agents_receiver = agents.rename(index=str, columns={'AgentId': 'ReceiverId'})
+  if receiver != 'All':
+    agents_receiver_ = agents_receiver.loc[lambda df: df.Prototype == receiver,:]
+    # check if receiver exists
+    if agents_receiver_.empty:
+      print("unknown Receiver, available Receiver are:")
+      for receiver_name in agents_receiver.Prototype.unique(): #check if the loop is correct should it not be trans.XX.XX
+        print(receiver_name)
+    else:
+      agents_receiver = agents_receiver_
+  # get sender
+  agents_sender = agents.rename(index=str, columns={'AgentId': 'SenderId'})
+  if sender != 'All':
+    agents_sender_ = agents_sender.loc[lambda df: df.Prototype == sender,:]
```

Comment:
This should get a better name than just an easy to mistype/misread trailing underscore.